src/dbally/views/structured.py

```diff
@@ -0,0 +1,100 @@
+import abc
+from typing import List
+
+from dbally.audit.event_tracker import EventTracker
+from dbally.data_models.execution_result import ViewExecutionResult
+from dbally.iql import IQLError, IQLQuery
+from dbally.iql_generator.iql_generator import IQLGenerator
+from dbally.llm_client.base import LLMClient
+from dbally.views.exposed_functions import ExposedFunction
+
+from .base import BaseView
+
+
+class BaseStructuredView(BaseView):
```

Comment:
Why didn't we use the [prefix][base class name] convention?

It's totally fine for me to keep the current name, but that was the first thing I thought about :smile:

Reply:
I'm not familiar with the convention. I try to name each class in a way that best describes the actual class.